store.php

```diff
@@ -1,16 +1,18 @@
 <?php
 
 // ファイルの読み込み
-require_once('./Models/Task.php');
+require_once('Models/Task.php');
+
 // データの受け取り
-// スーパーグローバル変数
 $title = $_POST['title'];
 $contents = $_POST['contents'];
-$time = date("Y/m/d H:i:s");
+$time = date("Y/m/d H:i:m");
```

Comment:
ここ授業で私が1度書き間違えしてしまったところなんですが、
秒数のところはsecondsなのでsです！ 🙏 
```suggestion
$time = date("Y/m/d H:i:s");
```